engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-packages.html

```diff
@@ -25,8 +25,13 @@ <h3 translate>Packages in {{ repository.name }}</h3>
   </div>
 
   <div data-block="actions">
+    <div bst-modal="removeContent()">
+      <div data-block="modal-header" translate>Remove {{ detailsTable.numSelected  }} Packages?</div>
+      <div data-block="modal-body" translate>Are you sure you want to remove the {{ detailsTable.numSelected }} packages selected?</div>
```

Comment:
For these do we need to use the plural feature that translate gives us?   Its annoying to use, but it makes it look much better than seeing '1 packages'